### PR TITLE
Use separate element color in theming

### PR DESCRIPTION
**Pull request by juliusknorr**
This way we can use a grey color when the primary color is to bright

fix for #5654

### CODE REVIEW

**Comment by MorrisJobke** [comment]
But now the background on the login screen is grey as well:

bildschirmfoto 2017-08-14 um 14 33 58

**Comment by juliushaertl** [comment]
@MorrisJobke Oh to much search/replace. Fixed that as well as the primary button.

**Comment by juliushaertl** [comment]
Fixed the tests and rebase to master. I've also adjusted the theming logic to match the new login design.

2017-09-01-232702_940x888_scrot

**Comment by dartcafe** [comment]
All elements seem to be visible when using a white theme, exept the default logo. It is still white on a white background on the login screen as in the menu. 
grafik
grafik

**Comment by MorrisJobke** [comment]
@dartcafe this is intended, because we only ship the white color of the icon. If you want a black one: upload that version of the icon

**Comment by MorrisJobke** [comment]
Rebased to fix some conflicts

**Review by MorrisJobke — APPROVED** [review]
Tested and works nicely 👍

**Comment by MorrisJobke** [comment]
@juliushaertl Could you open a backport PR for this one for stable12? :)

**Review by rullzer — APPROVED** [review]

**Comment by MorrisJobke** [comment]
> @juliushaertl Could you open a backport PR for this one for stable12? :)

Let me do this.

File: lib/composer/composer/autoload_classmap.php [hunk 1]
```diff
@@ -385,6 +385,7 @@
     'OC\\Command\\CallableJob' => $baseDir . '/lib/private/Command/CallableJob.php',
     'OC\\Command\\ClosureJob' => $baseDir . '/lib/private/Command/ClosureJob.php',
     'OC\\Command\\CommandJob' => $baseDir . '/lib/private/Command/CommandJob.php',
+    'OC\\Command\\CronBus' => $baseDir . '/lib/private/Command/CronBus.php',
```
Comment (nickvergessen) [inline lib/composer/composer/autoload_classmap.php]: 🙈 dont backport this one please

Comment (MorrisJobke) [reply lib/composer/composer/autoload_classmap.php]: It isn't: #6359

**Pull request merged**